.github/workflows/ci_cd.yml

```diff
@@ -1262,6 +1262,22 @@ jobs:
           wait_time: 60 # check every minute
           max_time: 1800 # allow up to 30 minutes for a deployment
 
+      - name: Trigger staging k6 load test
+        uses: convictional/trigger-workflow-and-wait@v1.6.5
+        with:
+          owner: WordPress
+          repo: openverse
+          token: ${{ secrets.ACCESS_TOKEN }}
+          workflow_file_name: k6.yml
+          wait_interval: 60
+          client_payload: |
+            {
+              "namespace": "frontend",
+              "scenario": "all",
+              "service_url": "https://staging.openverse.org/",
+              "report": true
+            }
```

Comment:
This "propagates failures" so if the load testing workflow fails, it will mark the staging deployment as failed (which will send a note in our alerts channel about a failed CI/CD run).

This might prove to be a mistake, if these tests are flaky in any way. But, we can try it for now, and easily add the `propagate_failure: false` option to the action.

We can also choose not to propagate the failure and instead use Grafana's alerting system to send a message when a load test run is uploaded with bad stats (failures).